test/e2e/test.utils.js

```diff
@@ -27,4 +26,53 @@ export const phrasesList = {
   },
 };
 
+export const networks = {
+  EMERYNET: 'emerynet',
+  LOCAL: 'local',
+};
 export const econGovURL = 'https://econ-gov.inter.trade/?agoricNet=local';
+
+export const AGORIC_NET = Cypress.env('AGORIC_NET') || 'local';
+
+export const DEFAULT_TIMEOUT =
+  AGORIC_NET === networks.LOCAL ? 1 * 60 * 1000 : 3 * 60 * 1000;
+export const DEFAULT_TASK_TIMEOUT =
+  AGORIC_NET === networks.LOCAL ? 1 * 60 * 1000 : 3 * 60 * 1000;
```

Comment:
nit: using `MINUTE_MS` instead of 60 * 1000 would make things clearer everywhere

Reply:
It's `1 * 60 * 1000` ........... I feel it's more semantic and clear.